salt/metalk8s/addons/logging/loki/deployed/datasource.sls

```diff
@@ -24,3 +33,11 @@ Deploy ConfigMap for Loki datasource:
               access: proxy
               url: http://loki.metalk8s-logging.svc:3100/
               version: 1
+{%- for index in range(loki.spec.deployment.replicas) %}
+            - name: Loki-{{ index }}
+              uid: metalk8s-loki-{{ index }}
+              type: loki
+              access: proxy
+              url: http://loki-{{ index }}.metalk8s-logging.svc:3100/
+              version: 1
+{%- endfor %}
```

Comment:
Wondering if we can have one "default" datasource per type or if it's global, maybe we want to set the "Loki" datasource as default ?

Reply:
Not possible, we can only set one default per org

Reply:
:cry: